tests/integ/modin/groupby/test_groupby_apply.py

```diff
@@ -857,6 +858,28 @@ def operation(df):
                 operation,
             )
 
+    # TODO: SNOW-1429855 test support include_groups = False
+    @sql_count_checker(
+        query_count=8,
+        udtf_count=UDTF_COUNT,
+        join_count=JOIN_COUNT,
+    )
+    def test_group_apply_return_df_from_lambda(self):
+        diamonds_path = (
+            pathlib.Path(__file__).parent.parent.parent.parent
+            / "resources"
+            / "diamonds.csv"
+        )
+        diamonds_pd = native_pd.read_csv(diamonds_path)
```

Comment:
can we use a smaller dataframe created inline that reproduces the error? I think that would be easier to maintain. Does the data have to come from a CSV?